.github/instructions/mastg-test.instructions.md

```diff
@@ -0,0 +1,217 @@
+MASTG-TEST-0204.md
+MASTG-TEST-0205.md
+```
+
+Example tests for reference:
+
+- [MASTG-TEST-0207](https://mas.owasp.org/MASTG/tests-beta/android/MASVS-STORAGE/MASTG-TEST-0207/)
+- [MASTG-TEST-0216](https://mas.owasp.org/MASTG/tests-beta/android/MASVS-STORAGE/MASTG-TEST-0216/)
+- [MASTG-TEST-0263](https://mas.owasp.org/MASTG/tests-beta/android/MASVS-STORAGE/MASTG-TEST-0263/)
+
+Notes:
+
+- Tests with `platform: network` are still organized under the OS folder that the MASVS category belongs to (for example, Android network tests live under `tests-beta/android/MASVS-NETWORK/`).
+
+Each test has two parts: the [Markdown metadata](#markdown-metadata) (YAML `front matter`) and the [Markdown body](#markdown-body).
+
+### Markdown: Metadata
+
+#### title
+
+Test titles should be concise and clearly state the purpose of the test.
+
+In some cases, the test name and the weakness may have the same title, but typically, tests cover different aspects of a weakness. Titles should reflect that.
+
+Avoid including Android or iOS unless necessary, as in "Insecure use of the Android Protected Confirmation API".
+
+Follow a consistent style across all test titles.
+
+**Conventions**
+
+- Static: “References to…” (semgrep/r2)
+- Dynamic: “Runtime Use …” (frida)
+
+Exceptions may apply where "Runtime ..." feels forced, for example, tests using adb, local backups, or filesystem snapshots.
+
+#### platform
+
+The mobile platform. One of the following: iOS, Android, or network.
+
+- Use network for platform-agnostic traffic analysis tests where the checks are performed purely on captured/observed traffic (often paired with type: [network]).
+
+#### id
+
+The test ID.
+
+#### weakness
+
+The MASWE weakness ID associated with this test.
+
+- In YAML front matter, specify the bare identifier (for example, `weakness: MASWE-0069`). In body text, include the leading `@` (for example, @MASWE-0069).
+
+#### type
+
+One or more test types.
+
+Supported:
+
+- `static`: analysis of the app binary, reverse-engineered source code, or developer artifacts that are available in the APK/IPA app package (e.g., Android manifest, Info.plist, entitlements, etc.). No execution of the app is required.
+- `dynamic`: analysis of the app while it is running and involves runtime analysis such as hooking or method tracing.
+- `manual`: manual steps that require human judgment, such as inspecting app behavior, UI, or configuration. This may include reverse engineering or runtime analysis that cannot be fully automated.
+- `network`: analysis of network traffic, while the app is running. Done externally, for example, using a proxy or network capture tool.
+- `filesystem`: analysis of the app's file system, including local storage or backups, which doesn't involve runtime analysis such as hooking or method tracing.
+- `developer`: tests only the developer can perform because they require access to the source code, build process, or other internal resources.
```

Comment:
Suggestion by @TheDauntless 

```suggestion
- `source-code`: tests only the developer can perform because they require access to the source code, build process, or other internal resources.
```